### PR TITLE
[enterprise-4.14] TELCODOCS-1958: Adding content for improved SR-IOV Network Operator node draining

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/TELCODOCS-1958

Link to docs preview:

- https://79809--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-networking-sriov-draining
- https://79809--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-sriov-device#configure-sr-iov-operator-parallel-nodes_configuring-sriov-device

QE review:
- [x] QE has approved this change.

Additional information:
Cherrypicked from the original main and RN PRs:
* [Main](https://github.com/openshift/openshift-docs/pull/74464)
* [RN](https://github.com/openshift/openshift-docs/pull/75620)

### CODE REVIEW

**Comment by ealcaniz** [comment]
LGTM

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -894,6 +894,13 @@ You can use egress IPs addresses on additional network interfaces as a Technolog
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/configuring-egress-ips-ovn.adoc#nw-egress-ips-multi-nic-considerations_configuring-egress-ips-ovn[Considerations for using an egress IP on additional network interfaces].
 
+[id="ocp-4-14-networking-sriov-draining"]
+==== Parallel node draining during SR-IOV network policy updates
+
+With this update, you can configure the SR-IOV Network Operator to drain nodes in parallel during network policy updates. The option to drain nodes in parallel enables faster rollouts of SR-IOV network configurations. You can use the `SriovNetworkPoolConfig` custom resource to configure parallel node draining and define the maximum number of nodes in the pool that the Operator can drain in parallel.
+
+For further information, see xref:../networking/hardware_networks/configuring-sriov-device.adoc#configure-sr-iov-operator-parallel-nodes_configuring-sriov-device[Configuring parallel node draining during SR-IOV network policy updates].
```
Comment (lahinson) [inline release_notes/ocp-4-14-release-notes.adoc:902]:
```suggestion
For more information, see xref:../networking/hardware_networks/configuring-sriov-device.adoc#configure-sr-iov-operator-parallel-nodes_configuring-sriov-device[Configuring parallel node draining during SR-IOV network policy updates].
```
Using "For more information" per the OCP guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentences

**Review by lahinson — COMMENTED** [review]
Nice work on this! I left some comments for your consideration.

**Comment by stevsmit** [comment]
This LGTM. Sometimes  "an sriov. . ." is used and other times "a sriov. . ." but that's not enough to hold up a merge review.

**Pull request merged**